operating/verify-image-signatures.adoc

```diff
@@ -13,18 +13,30 @@ You can also enforce the policy by using the {product-title-short} admission con
 
 [NOTE]
 ====
-* {product-title-short} only supports Cosign signatures and Cosign Public Keys/Certificates verification. For more information about Cosign, see link:https://docs.sigstore.dev/cosign/overview[Cosign overview].
-* For Cosign signature verification, {product-title-short} does not support communication with the transparency log link:https://docs.sigstore.dev/logging/overview/[Rekor].
+* {product-title-short} supports Cosign signature verification by using Cosign public keys, Cosign certificates, or both.
++
+For more information about Cosign, see link:https://docs.sigstore.dev/cosign/signing/overview/[Overview] (Sigstore documentation).
+* For Cosign signature verification, {product-title-short} supports communication with the transparency log. 
++
+For more information, see link:https://docs.sigstore.dev/logging/overview/[Rekor] (Sigstore documentation).
+* For Cosign signature verification, {product-title-short} supports keyless verification of the signatures by using {rh-rhtas-first}.
```

Comment:
This makes it sound like we only support keyless in combination with RHTAS, which is not correct. Using RHTAS is just one option of implementing keyless verification (namely when customers want to self-host the key infratructure instead of using the public Sigstore one).